tests/cached-document-loader.js

```diff
@@ -0,0 +1,46 @@
+import contexts from '../src/utils/vc/contexts';
```

Comment:
The test for private delegation required a document loader. I pulled the `documentLoader` out of `claim-deduction.test.js`, so it could be used by the other tests.